pkg/server/image_status.go

```diff
@@ -49,6 +64,45 @@ func (c *criContainerdService) ImageStatus(ctx context.Context, r *runtime.Image
 	}
 	runtimeImage.Username = username
 
-	// TODO(mikebrow): write a ImageMetadata to runtime.Image converter
-	return &runtime.ImageStatusResponse{Image: runtimeImage}, nil
+	return runtimeImage
+}
+
+// toCRIImageInfo converts internal image object information to CRI image status response info map.
+func (c *criContainerdService) toCRIImageInfo(ctx context.Context, image *imagestore.Image, verbose bool) map[string]string {
+	if !verbose {
+		return nil
+	}
+
+	info := make(map[string]string)
+	i := image.Image
+
+	// TODO (mikebrow): discuss predefining constants for some or all of these key names in CRI for these info map values
+	info["imageConfig"] = marshallToString(image.Config)
+
+	descriptor, err := i.Config(ctx)
+	if err != nil {
+		info["imageConfigDescriptor"] = err.Error()
```

Comment:
I think we can rename the name corresponding with OCI, like:
`imageConfig`                    -> `config`
`imageConfigDescriptor`   -> `configDescriptor`
`imageTargetDescriptor`   -> `manifestoDescribor`
`imageTargetDigestsInfo`  -> `layerInfo`

wdyt?